app/workers/deletes.php

```diff
@@ -113,6 +113,8 @@ public function run(): void
                 Console::error('No delete operation for type: ' . $type);
                 break;
         }
+
+        Authorization::reset();
```

Comment:
Why did we change the Authorization behavior? usually in worker context we want to act as admin and have no permission check.

Reply:
Initially, I wanted to only disable it at the top of the file, but that didn't work for some reason - some other files must have reseted it somewhere.

What I did is I copied the solution from another worker (certificates), to disable at the beginning of the run, and reset at the end.

Reply:
Maybe there is some calls in the database library itself?

Reply:
Maybe the db library itself has calls to reset it?

Reply:
Maybe the db library itself has calls to reset it?

Reply:
A problem was caused when library called reset, yes. It is now fixed, please revisit.